src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+
+export const TabsPage: React.FC = () => {
+  const { tabId } = useParams();
+  const tabs = [
+    { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+    { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+    { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+  ];
```

Comment:
Move such constants outside component, we definitely don't want to create new array on every component render